set-recording-comments.user.js

```diff
@@ -11,6 +11,7 @@
 // @match          *://*.mbsandbox.org/release/*
 // @exclude        *musicbrainz.org/release/*/*
```

Comment:
Could you update this one so all expressions are similar? (adding prefix `*://`)

Reply:
Sorry, I just remembered that the `@match` patterns do not work for `@exclude` rules. You either have to use the rule as it already is or use `@exclude-match`. Please do not merge yet, I will fix it again!

Reply:
I decided to go with `@exclude *musicbrainz.org/release/add*` which will also match possible query parameters while `@exclude-match *://*.musicbrainz.org/release/add` would have been cleaner but unfortunately incompatible with Greasemonkey.

Reply:
Good catch and well done! :+1: 
Is this ready to be merged? It seems so, but I'd rather ask to be sure.

Reply:
Yes it is 😇